### PR TITLE
Simplest representations for a database View. I'm also cool with

### PR DESCRIPTION
**Pull request by vivekmenezes**
checking this into an experimental directory.

### CODE REVIEW

File: proto/view.proto [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+// ResultSet is a slice of a database View. A View of the database can be
+// very large and is to be accommodated in a stream of ResultSet
+message ResultSet {
+  // The columns that form the view.
+  repeated Column columns = 1;
+  // Some of the results in a View. All the results are for the same columns.
+  repeated Result results = 3;
+}
```
Comment (petermattis) [inline proto/view.proto]: This (`Column`, `Result` and `ResultSet`) feels a bit premature. It isn't needed for a "key-value" table. And we first need to iron out the `NamespaceDescriptor` and `TableDescriptor`. Perhaps I'm missing some context on why this comes first.

Comment (vivekmenezes) [inline proto/view.proto]: I'd like us to iron out the protobuf API to be as close to the final
outcome so we don't have to keep changing all the key value clients
significantly
every time we change this API. Besides it's nice to see the compromises we
need to make on behalf of a key value only client when they use a view
approach.

We do want ResultSet to be part of the response sent back by Cockroach
right?

Comment (petermattis) [inline proto/view.proto]: Sent back by Cockroach when? What is a `View`? There seems to be a gap in the discussion between the doc you were working on yesterday and this change.

Comment (vivekmenezes) [inline proto/view.proto]: Okay let me write up a little more on this in the design doc and we'll take
it from there. I'll delete this pull request for now.

**Pull request closed without merging**